frontend/src/metabase/ui/components/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -81,6 +87,43 @@ export function MultiAutocomplete({
   onOptionSubmit,
   ...otherProps
 }: MultiAutocompleteProps) {
+  const tc = useTranslateContent();
+  const sortByTranslation =
+    PLUGIN_CONTENT_TRANSLATION.useSortByContentTranslation();
+
+  const sortedData = useMemo(() => {
```

Comment:
I followed the pattern from `SingleSelectListField.tsx` . It's not pretty, but the sorting needs to happen as close to where the options are used as possible